docs/guides/wifi.md

```diff
@@ -13,57 +13,66 @@ Refer to the "Updating Kernel" section on your distro's FAQ for instructions if
 - [Manjaro](https://wiki.t2linux.org/distributions/manjaro/faq/#updating-kernel)
 - [Ubuntu](https://wiki.t2linux.org/distributions/ubuntu/faq/#updating-kernel)
 
-## Getting the firmware
+## Setting up
 
-In order to get wifi running on Linux, we need to get the firmware from macOS and copy it to Linux with appropriate changes. Thanks to Asahi Linux, who has made the process much simpler. You may follow the steps below to get the firmware and copy it to Linux properly.
+We now use a script which can help you set up Wi-Fi. Follow the instructions below to use this script :-
 
-1. Boot into macOS.
+### On macOS
 
-2. Open the terminal and run :-
+1. Click [here](https://github.com/t2linux/wiki/tree/master/tools/wifi.sh) to download the script.
```

Comment:
Can we use a relative file path instead? That should make it work offline and on different branches

Reply:
For eg?

Reply:
check 3894fcc @networkException